src/point_vector_fallbacks.jl

```diff
@@ -164,66 +167,30 @@ macro default_manifold_fallbacks(TM, TP, TV, pfield::Symbol, vfield::Symbol)
             return X
         end
     end
-
-    for BT in [
-        ManifoldsBase.DISAMBIGUATION_BASIS_TYPES...,
-        ManifoldsBase.DISAMBIGUATION_COTANGENT_BASIS_TYPES...,
-    ]
-        push!(
-            block.args,
-            quote
-                function ManifoldsBase.get_coordinates!(M::$TM, Y, p::$TP, X::$TV, B::$BT)
-                    return get_coordinates!(M, Y, p.$pfield, X.$vfield, B)
-                end
-
-                function ManifoldsBase.get_vector(M::$TM, p::$TP, X, B::$BT)
-                    return $TV(get_vector(M, p.$pfield, X, B))
-                end
-
-                function ManifoldsBase.get_vector!(M::$TM, Y::$TV, p::$TP, X, B::$BT)
-                    return get_vector!(M, Y.$vfield, p.$pfield, X, B)
-                end
-            end,
-        )
-    end
-
-    for VTM in [ParallelTransport, VECTOR_TRANSPORT_DISAMBIGUATION...]
-        push!(
-            block.args,
-            quote
-                function ManifoldsBase.vector_transport_direction!(
-                    M::$TM,
-                    Y::$TV,
-                    p::$TP,
-                    X::$TV,
-                    d::$TV,
-                    m::$VTM,
-                )
-                    vector_transport_direction!(
-                        M,
-                        Y.$vfield,
-                        p.$pfield,
-                        X.$vfield,
-                        d.$vfield,
-                        m,
-                    )
-                    return Y
-                end
-                function ManifoldsBase.vector_transport_to!(
-                    M::$TM,
-                    Y::$TV,
-                    p::$TP,
-                    X::$TV,
-                    q::$TP,
-                    m::$VTM,
-                )
-                    vector_transport_to!(M, Y.$vfield, p.$pfield, X.$vfield, q.$pfield, m)
-                    return Y
-                end
-            end,
-        )
+    for f_postfix in [:default, :orthogonal, :orthonormal, :vee, :cached, :diagonalizing]
+        ca = Symbol("get_coordinates_$(f_postfix)")
+        cm = Symbol("get_coordinates_$(f_postfix)!")
+        va = Symbol("get_vector_$(f_postfix)")
+        vm = Symbol("get_vector_$(f_postfix)!")
+        push!(block.args, quote
+            import ManifoldsBase: $ca, $cm, $va, $vm
```

Comment:
I wouldn't import these symbols in a macro, but rather do it like `function ManifoldsBase.($ca)(M::$TM, p::$TP, X::$TV, B)`

Reply:
Hm, it took me literally _hours_ to get this working, so feel free to improve this, I think I was at your suggestion once and it did not work at all, maybe the function definition worked, but it was not callable - or even the function definition did not work.

But I am also far away from being an expert on quoted and escaped stuff. I just did guesswork until it worked.

Reply:
OK, I've fixed it myself.